samcli/commands/local/start_api/cli.py

```diff
@@ -61,6 +61,12 @@
     default="public",
     help="Any static assets (e.g. CSS/Javascript/HTML) files located in this directory " "will be presented at /",
 )
+@click.option(
+    "--disable-authorizer",
+    is_flag=True,
+    default=False,
+    help="Disable custom authorizer from being invoked. This is useful for local testing."
```

Comment:
```suggestion
    help="Disable custom Lambda Authorizers from being parsed and invoked."
```
Thoughts about leaving the local testing line out? Mainly since this is `local start-api`, so "local testing of xyz" is sort of what is expected out of running this command :)